archivist_samples/door_entry/run.py

```diff
@@ -48,9 +48,6 @@
 
 BEHAVIOURS = [
     "Attachments",
```

Comment:
Is the Attachments behaviour still necessary? Can't you achieve that with `event.asset_attributes.attachments`?

Reply:
well the doors do have attachments and the attachments behaviour is still valid?